src/test/java/org/broadinstitute/consent/http/service/dao/DataAccessRequestServiceDAOTest.java

```diff
@@ -152,7 +152,7 @@ private DataAccessRequest createDataAccessRequest(Integer userId, Integer collec
       String darCode) {
     DataAccessRequestData data = new DataAccessRequestData();
     data.setProjectTitle("Project Title: " + RandomStringUtils.random(50, true, false));
-    data.setDarCode(darCode);
+//    data.setDarCode(darCode);
```

Comment:
Commented code.

It did make me wonder if we ever take this value from user input and use it for something.

Reply:
My fixes to these test classes keep getting overwritten from merges - I need to review this class again.

Reply:
Setting `darCode` on the data object is how we used to store the source of truth for this value. When we implemented `DarCollection`, we moved it there so that was the new source of truth. Tests, however, lagged behind.